zerver/lib/test_classes.py

```diff
@@ -2001,6 +2001,96 @@ def capture_send_event_calls(
 
         self.assert_length(lst, expected_num_events)
 
+    @override
+    def send_personal_message(
+        self,
+        from_user: UserProfile,
+        to_user: UserProfile,
+        content: str = "test content",
+        *,
+        read_by_sender: bool = True,
+        skip_capture_on_commit_callbacks: bool = False,
```

Comment:
I think we probably need a docstring or something that explains how a caller should decide whether to pass `skip_capture_on_commit_callbacks` ... possibly something brief linking to a new section in `docs/subsystems/events-system.md`

Reply:
Added docstring.

I think it's not a big enough change to highlight in docs?

Reply:
Yeah I think the new section would be on capturing on-commit callbacks in general, not this override.